addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -643,6 +643,30 @@ export class MassMailingHtmlField extends HtmlField {
             this.props.record.data[this.props.name] = this.props.record.data.body_html;
         }
         await super._setupReadonlyIframe();
+
+        const iframeTarget = this.sandboxedPreview
+            ? this.iframeRef.el.contentDocument.documentElement
+            : this.iframeRef.el.contentDocument.querySelector("#iframe_target");
+        this._fixInlines(iframeTarget);
+    }
+    _fixInlines(subRoot) {
```

Comment:
This made sense in the `qweb_plugin`, but here in the html_field I think we could rename this function `_fixInlineDynamicPlaceholders`